packages/apps/package.json

```diff
@@ -23,8 +23,8 @@
     "lint:ts": "tslint -c tslint.json -p ."
   },
   "devDependencies": {
-    "@counterfactual/contracts": "0.1.5",
```

Comment:
Why two versions?

Reply:
The apps package was already behind before this PR by one version.